src/Dropzone/Resources/assets/src/controller.ts

```diff
@@ -42,31 +42,34 @@ export default class extends Controller {
         this.previewImageTarget.style.display = 'none';
         this.previewImageTarget.style.backgroundImage = 'none';
         this.previewFilenameTarget.textContent = '';
+        document.querySelectorAll('.dropzone-preview-image-container').forEach((e) => e.remove());
```

Comment:
Sorry if I'm missing it, where/what is creating this `.dropzone-preview-image-container` element?